orca-peering/README.md

```diff
@@ -135,3 +136,4 @@ The following dynamic properties are exposed and can be controlled at runtime vi
 |----------|---------|-------|
 |`pollers.peering.enabled`          | `true` | if set to `false` turns off all peering |
 |`pollers.peering.<PEERID>.enabled` | `true` | if set to `false` turns off all peering for peer with given ID | 
+|`pollers.peering.max-allowed-delete-count`| 1000 | maximum number of executions that are allowed to be deleted in a single run of the agent |
```

Comment:
It might be interesting to discuss that default value a little. I guess for the most part that's the volume of deletions we might expect from one of the cleanup agents at Netflix scale. But one could imagine that dropping 1000 executions on a smaller Spinnaker installation is dropping the entire thing, so how big 1000 is relative